src/components/Header/HeaderModal.jsx

```diff
@@ -1,40 +1,51 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { HeaderButton } from './Header.styled';
 import HeaderSVGPhoto from '../../images/svg/svgheader/HeaderSVGPhoto';
 import HeaderButtonSVG from '../../images/svg/svgheader/HeaderButtonSVG';
 
 import {
   Backdrop,
-  ButtonCancel,
   ButtonLogOut,
   DivButtonLogOut,
   DivHeaderModalContainer,
-  DivTest,
   HeaderModalButton,
   HeaderModalButtonSpan,
   HeaderModalContainer,
   HeaderModalLogOutContainer,
   SpanLogOut,
   SpanLogOutQuestion,
+  Test1,
+  UserIMG,
 } from './HeaderModalStyled';
 import SettingModal from '../SettingModal/SettingModal';
 import SettingSVG from '../../images/svg/svgheader/SettingSVG';
 import ClouseSVG from '../../images/svg/svgheader/ClouseSVG';
 import HeaderButtonRotateSVG from '../../images/svg/svgheader/HeaderButtonRotateSVG';
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { logoutThunk } from '../../store/auth/thunks';
 import { useNavigate } from 'react-router';
 import { toast } from 'react-toastify';
+import { selectUser } from '../../store/auth/selectors';
 
 const HeaderModal = () => {
   const [isModalOpen, setIsModalOpen] = useState(false);
   const [isHeaderModalOpen, setIsHeaderModalOpen] = useState(false);
   const [isHeaderModalLogOut, setIsHeaderModalLogOut] = useState(false);
   const [isBackdropVisible, setIsBackdropVisible] = useState(false);
+  const [imageUser, setImageUser] = useState('');
+  const user = useSelector(selectUser);
```

Comment:
could you pls rename this variable 'user' into for ex userProfile or userObject - that will help our colleagues in future to avoid misspelling and possible mistakes